app/controllers/organisations.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class organisations extends Controller {
+
+    function __construct() {
+        $this->load_model('organisations_model');
+    }
+
+    function index() {
+        $this->load_view('organisations/index');
+    }
+
+    function create() {
+        $created = null;
+
+        if (isset($_POST['submit'])) {
+            $created = $this->organisations_model->create_org(
+                $_POST['org'],
+                $_POST['contact']
+            );
+        }
+
+        $this->load_view('organisations/create', array(
+            'created' => $created
+        ));
+    }
+
+    function org_list() {
```

Comment:
I'm sorry. `list` won't work. If we want we can set routes for it.. Or have some other better name.